ambuda/views/proofing/main.py

```diff
@@ -156,3 +156,16 @@ def recent_changes():
         session.query(db.Revision).order_by(db.Revision.created.desc()).limit(100).all()
     )
     return render_template("proofing/recent-changes.html", revisions=recent_revisions)
+
+
+@bp.route("/talk")
+def talk():
+    """Show discussion across all projects."""
+    session = q.get_session()
+    projects = q.projects()
+
+    # FIXME: optimize this once we have a higher thread volume.
+    all_threads = [(p, t) for p in projects for t in p.board.threads]
```

Comment:
Just a quick optimization -- can we first query for all threads `q.threads()` (so we can limit this query for the X most recently created threads), then get each thread's project information?

Reply:
We can get `thread.board_id` then find the project that owns that `board_id` with a join, but it's a little tedious to do this the right way. For now the data volume is so low that it's not worth it.